tests/test_forest.py

```diff
@@ -107,11 +116,12 @@ def test_oob_score(make_whas500):
     assert round(abs(forest.oob_score_ - 0.753010685), 6) == 0.0
```

Comment:
I noticed that two unit tests `test_fit_predict` and `test_oob_score` directly put the expected results (for RandomSurvivalForest) in the assertion statement, and not sure how to modify it to adapt to `ExtraSurvivalTrees`. Will appreciate your suggestions very much ;-)

Reply:
You can pass in the expected value via pytest's parametrize too.